config/dependencies/quickstart.yaml

```diff
@@ -108,7 +108,7 @@ spec:
             - name: MINIO_SECRET_KEY
               value: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
           # image: quay.io/cloudservices/minio:latest
-          image: kserve/modelmesh-minio-examples:latest
+          image: kserve/modelmesh-minio-examples:v0.10.0
```

Comment:
@njhill -- you might have to push this unless some GitHub action does

Reply:
Alright, @yuzisun added me to the [`kserve` org on DockerHub](https://hub.docker.com/orgs/kserve/members) so I was able to push a new tag `v0.10.0` based on the `latest` image

```Bash
docker manifest create kserve/modelmesh-minio-examples:v0.10.0 kserve/modelmesh-minio-examples:latest
docker manifest push kserve/modelmesh-minio-examples:v0.10.0
```

https://hub.docker.com/r/kserve/modelmesh-minio-examples/tags

Reply:
thanks @yuzisun 👍🏻